### PR TITLE
fix(compartment-mapper): fix for bundling of appended cjs exports

### PR DESCRIPTION
**Pull request by kumavis**
bug fix for bundler for supporting the following commonjs exports patterns 
```js
module.exports.abc = 123
exports.xyz = 456
```

Originally tried to fix it with https://github.com/endojs/endo/commit/7daba15b88752898ab1f9a1895d5cf479ea3575f
but this was insufficient, need to add missing cells and update the `*` cell

not sure if this is the correct change or just a bandaid but it got tests passing

### CODE REVIEW

File: packages/compartment-mapper/test/test-bundle.js
Comment (kriskowal) [inline packages/compartment-mapper/test/test-bundle.js:111]: I suspect that awaiting here is sufficient since the intent is to stop the test if it throws. The subsequent errors will be less direct than failing here.

File: packages/compartment-mapper/test/test-bundle.js
Comment (kriskowal) [inline packages/compartment-mapper/test/test-bundle.js:106]: As a matter of tending toward web-portable ocap style, you could `await writePowers.write` here.

**Review by kriskowal — COMMENTED** [review]
I defer to @naugtur and I see nothing to block this from landing. Consider adding a note to `NEWS.md`.

File: packages/compartment-mapper/test/test-bundle.js
Comment (kumavis) [reply packages/compartment-mapper/test/test-bundle.js:106]: oh this was not supposed to make it in, thanks for catching it

File: packages/compartment-mapper/test/test-bundle.js
Comment (kumavis) [reply packages/compartment-mapper/test/test-bundle.js:111]: `compartment.evaluate(bundle)` returns synchronously and does not return a promise, so i'm confused about the purpose of the `await`

Comment (kriskowal) [reply packages/compartment-mapper/test/test-bundle.js:111]: my bad, just call :-)

File: packages/compartment-mapper/src/bundle-cjs.js
Comment (naugtur) [inline packages/compartment-mapper/src/bundle-cjs.js:19]: improvement for bundle size when minified :+1:

File: packages/compartment-mapper/src/bundle-cjs.js
Comment (naugtur) [inline packages/compartment-mapper/src/bundle-cjs.js:35]: I mentioned earlier that in regular endo apps the `*` behavior is implemented elsewhere, but that elsewhere doesn't exist in bundler, so I think this is the right place to do this.

File: packages/compartment-mapper/test/test-bundle.js
Comment (naugtur) [inline packages/compartment-mapper/test/test-bundle.js:112]: if required in cjs they should not expose the default unless it was explicitly assigned by the module. I think that's true with the current implementation. Not sure if there's a fixture for that, but there might already be.

**Review by naugtur — APPROVED** [review]

File: packages/compartment-mapper/test/test-bundle.js
Comment (kumavis) [reply packages/compartment-mapper/test/test-bundle.js:112]: im surprised `default` is not like a Symbol or something, but I dont know the desired behavior for compat

**Pull request merged**
